### PR TITLE
#159965492 Add pagination for articles

### PR DESCRIPTION
**Pull request by Quantum-35**
## What does this PR do?
```
- Adds Pagination support for articles.
```

## Description of Task to be completed?
```
Pagination support is lacking for our articles. Reading can be a nightmare when we have so many articles. 

```

## How should this be manually tested?

```
- Clone this repo and pip install requirements.txt
- Sign up and Sign in 
- Navigate to this endpoint http://127.0.0.1:8000/api/articles
- Post more the five articles and run get
- You will notice that only five articles are returned with next and previous 
   endpoints automatically generated

```

## What are the relevant pivotal tracker stories?

[#159965492](https://www.pivotaltracker.com/story/show/159965492)

## Screenshots if available to support PR.

screen shot 2018-09-11 at 16 03 11

## Questions:

```
Was this PR helpful?

```

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -153,7 +153,7 @@
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
-    'PAGE_SIZE': 10,
+    'PAGE_SIZE': 2,
```
Comment (Derrickkip) [inline authors/settings.py]: I suggest increasing the page size to five or above

**Review by Mutuba — COMMENTED** [review]
Increase the number of articles per page, have atleast 5.

**Comment by Quantum-35** [comment]
@Mutuba @Derrickkip I have implemented that review.

**Review by Derrickkip — DISMISSED** [review]
Looks Good

File: authors/settings.py (hunk 1 above)
Comment (Quantum-35) [reply authors/settings.py]: I have implemented that.

**Review by Mutuba — DISMISSED** [review]

**Review by Derrickkip — APPROVED** [review]

**Review by Mutuba — APPROVED** [review]

**Review by WNjihia — APPROVED** [review]

**Pull request merged**